styles.css

```diff
@@ -0,0 +1,707 @@
+@media screen and (min-width: 320px) {
+  /* RESETS */
+  * {
+    margin: 0%;
+    padding: 0%;
+    height: auto;
+    list-style: none;
+    box-sizing: border-box;
+  }
+
+  button {
+    border: none;
+  }
+
+  .heading-text {
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: clamp(40px, 48px);
+    line-height: 52px;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .description-text {
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  #main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  /* TOP NAVIGATION SECTION */
+  #top-navigation {
+    width: 100%;
+    height: 72px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: fixed;
+    background: none;
+  }
+
+  #logo {
+    position: absolute;
+    left: 10%;
+    top: 20px;
+  }
+
+  #navigation-list {
+    display: none;
+    flex-direction: row;
+    gap: 16px;
+    position: relative;
+    top: 16px;
+  }
+
+  .navigation-item {
+    list-style: none;
+    display: flex;
+    justify-items: center;
+    width: 67px;
+  }
+
+  .navigation-item-text {
+    width: 63px;
+    height: 20px;
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  #toolbar-menu-button {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    top: 2px;
+    border-radius: 8px;
+  }
+
+  #toolbar-menu-icon {
+    width: 24px;
+    height: 24px;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    top: 20px;
+    left: 8px;
+    border: none;
+  }
+
+  #toolbar-menu-union-icon,
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    background-color: #000;
+    position: absolute;
+    width: 100%;
+    height: 2px;
+  }
+
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    content: "";
+  }
+
+  #toolbar-menu-union-icon::before {
+    transform: translate(0, 10px);
+  }
+
+  #toolbar-menu-union-icon::after {
+    transform: translate(0, -10px);
+  }
+
+  /* Headline section */
+  #headline {
+    height: auto;
+    padding: 160px 0;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
```

Comment:
[OPTIONAL] Top quality styling over here but do consider this: It is always a good idea and good practice as well to style your tags using classes instead of ids. ids have extra underlying logic that, if not handled properly, can cause bugs as your project scales. I mark this as optional because it doesn't break your code but I urge you to pay special attention to this